src/test/java/de/tudarmstadt/ukp/jwktl/parser/de/DETranslationHandlerTest.java

```diff
@@ -224,6 +224,11 @@ public void testBoot() throws Exception {
 		assertEquals("{{f}}", trans.get(2).getAdditionalInformation());
 	}
 
+	public void testParsesVerkehrspaedagoge() throws Exception {
+		IWiktionaryPage page = parse("Verkehrspaedagoge.txt");
+		assertEquals(1, page.getEntryCount());
```

Comment:
I suggest to test at least for the number of translations parsed

Reply:
No translations are parsed, but we can check for `isEmpty()`.